encord/objects/ontology_labels_impl.py

```diff
@@ -2071,7 +2125,7 @@ def _parse_label_row_dict(self, label_row_dict: dict) -> LabelRowReadOnlyData:
             priority=label_row_dict.get("priority", self._label_row_read_only_data.priority),
             client_metadata=label_row_dict.get("client_metadata", self._label_row_read_only_data.client_metadata),
             images_data=label_row_dict.get("images_data", self._label_row_read_only_data.images_data),
-            file_type=label_row_dict.get("file_type", None),
+            file_type=file_type,
```

Comment:
For the longest time, this has just been `None` lolol.

Reply:
Yes your comment is correct. I'm unsure as to what you want me to do though.
Are you trying to suggest that we don't overwrite it? And keep what we have in file_type before calling initialise_labels?